libmscore/score.h

```diff
@@ -816,7 +823,10 @@ class Score : public QObject, public ScoreElement {
       void renderMidi(EventMap* events);
       void renderStaff(EventMap* events, Staff*);
       void renderSpanners(EventMap* events, int staffIdx);
-      int renderMetronome(EventMap* events, Measure* m, int playPos, int tickOffset, bool countIn);
+      void renderMetronome(EventMap* events, Measure* m, int tickOffset);
+
+      TimeSigFrac::BeatType tick2beatType(int tick);
+      NPlayEvent beatType2metronomeEvent(TimeSigFrac::BeatType beatType);
```

Comment:
This one doesn't really need to be in score.h right? It could be anywhere else and we would spare the include from synthesizer/event.h it could even be static.